### ISSUE TITLE
Fix local storage to save images for better usage

### ISSUE THREAD

**Issue by JoseDeFreitas** [JoseDeFreitas/thisisme#3]
_To know what I mean, go to the website [thisisme.vercel.app](https://thisisme.vercel.app) (which is the website of the application.) That build if from the last commit made._

As you see in the "Result" section, the image appears, but I want to layout them side by side (the full SVG image on the Result section is set 900 of width. Each image is 450 of width. I want to have 2 images per row and per column depending on what images the user filled up.) If a user doesn't fill the inputs of a section at all, the image should not be displayed in the "Result" section. See the [pages folder](https://github.com/JoseDeFreitas/thisisme/tree/main/src/pages) and the [result.js file](https://github.com/JoseDeFreitas/thisisme/blob/main/src/pages/Result.js) for more information.
If you want to work on it, mention me and I'll assign you. The PR you create will be reviewed.

**Mention me if you need more information**. Thanks in advance! 😄

**Comment by fernandoamz** [comment]
Hi @JoseDeFreitas, I could help you with this

**Comment by fernandoamz** [comment]
@JoseDeFreitas, I want to clarify what you mean. I think that you need something like this?

<img width="674" alt="Screen Shot 2020-10-03 at 8 04 59 PM" src="https://user-images.githubusercontent.com/7818620/95004670-e7224600-05b3-11eb-994a-20a5401da7c4.png">

**Comment by JoseDeFreitas** [comment]
Hello @fernandoamz! The full SVG image (the one in the Result section) is 900 width. The other images (of the other sections, such as Programming, Technologies, etc.) are 450 widths, so they can fit 2 of them in the full SVG image (in the Result section). The image attached here is how I like it to look like (but with the card styles and content, of course): https://imgur.com/xGEkRWl. _Don't mind the number of the cards in this image, it's just to show you. If a section inputs are not filled up the image should just not show up in the full SVG image (Result section)._

**Comment by fernandoamz** [comment]
@JoseDeFreitas, Yeah, assign me this. I will make the pull request as soon possible.

**Comment by alex-anhvuong** [comment]
Hi @JoseDeFreitas , I would like to help with this issue please.

**Comment by JoseDeFreitas** [comment]
Hello @alex-anhvuong! There's already an assigned person to this issue, but I'll assign you as well. This may be a big change so working more than 1 person would be good. Thank you for your offering. Any question just mention me.

**Comment by JoseDeFreitas** [comment]
Hello! There have been some important changes recently. These should not break what you're doing, but it's important that you pull them. Please see the [latest Pull requests](https://github.com/JoseDeFreitas/thisisme/pulls) (both closed and opened.) Let me know anything. Thank you!

**Comment by alex-anhvuong** [comment]
Hi @JoseDeFreitas,
I've made a pull request for this (#28).  Please have a check. Thank you.

**Comment by chris--jones** [comment]
@JoseDeFreitas 
I was messing around with this and ended up refactoring too much and decided I'd just make my own fork without all the things I didn't like: https://github.com/chris--jones/thisisme - feel free to check it out and take anything you like 😄

There's also a demo running here: https://chris--jones.github.io/thisisme

**Comment by JoseDeFreitas** [comment]
Hello @chris--jones! I just saw your repository. Looks really nice. You make it a lot better 😅 indeed a better approach. I'll look deeper into it when I get to work on this project. Thank you so much 👍.
